.github/workflows/rust-ci.yml

```diff
@@ -55,6 +55,8 @@ jobs:
         with:
           submodules: true
       - uses: EmbarkStudios/cargo-deny-action@v1
+        with:
+          command-arguments: -D warnings
```

Comment:
Saw some warnings locally, let's see if this catches them :)

Reply:
https://github.com/EmbarkStudios/sentry-contrib-rust/actions/runs/3491841510/jobs/5844941478 Perfect!